File: Fisica/src/Fisica/Models/AulaModel.cs
namespace Fisica.Models
{
    public class AulaModel
    {
        public long Id { get; set; }
        public string Titulo { get; set; }
        public string Descricao { get; set; }
        public string AreaFisica { get; set; }
        public string ProfessorNome { get; set; }
        public long ProfessorId { get; set; }
        public IList<SessaoAulaModel>? Sessoes { get; set; }
        public IList<ComentarioAulaModel>? Comentarios { get; set; }
    }
}

File: Fisica/src/Fisica.Website/ClientApp/src/app/shared/models/aula.model.ts
import { SessaoAula } from "./sessao-aula.model";

export class Aula {
  constructor(
    public id?: number,
    public titulo?: string,
    public descricao?: string,
    public areaFisica?: string,
    public professorNome?: string,
    public professorId?: number,
    public sessoes?: SessaoAula[]
  ) { }
}
